### DISCUSSION TITLE
v0.4.23

### DISCUSSION THREAD

**Discussion by tefkah** [tefkah/zotero-night#125] [Announcements]
Fixes #76, #87.

It also actually makes the default pdf filter setting do something.

This is probably the last release for Zotero 6, I will be focusing on Zotero 7 compatibility moving forward. I will try to make it work with 6, but no guarantees!

<hr /><em>This discussion was created from the release <a href='https://github.com/tefkah/zotero-night/releases/tag/'>v0.4.23</a>.</em>

**Comment by pbk0** [comment]
I tried to install, but I got this error for Zotero 6 on windows 
![image](https://github.com/tefkah/zotero-night/assets/2289914/c36a1551-1b19-4d7e-a49e-7547c7547e35)

**Reply by juangimenezg** [reply]
You can fix it by doing the following (Windows):

Short fix: 
= Need to remove the "release" folder, and place all children folders and files at the top of the xpi file. You can use 7-Zip.


Detailed fix:
= Open night.xpi using 7-Zip
= Extract everything to a local folder
= Locate yourself inside the "release" folder. You will see 4 folders and 2 files
= Select those 4 folders and 2 files, and use 7-Zip to create a file "night_new.xpi", using: archive format: zip, compression normal, deflate, 32kb, 32
= Once you have the file ready: drag to  Add-ons windows inside Zotero

**Reply by RaymondKJ** [reply]
sorry, it doesn't work.

**Reply by tefkah** [reply]
which version of zotero are you on

**Reply by tefkah** [reply]
So sorry, @juangimenezg is right, I incorrectly packed the .xpi myself manually because the automatic release wasn't working.

Try downloading again, it should work now!

**Reply by RaymondKJ** [reply]
@tefkah It works now, thank you!

**Comment by donkikos** [comment]
Getting the same error on Mac. 

```
Add-on Installation Failed

The add-on "%S" could not be installed. It may be incompatible with this version of Zotero.
```

The solution mentioned by @juangimenezg did not work. The `release` folder is empty.

**Reply by tefkah** [reply]
Hm, weird, let me have a look

**Reply by tefkah** [reply]
Their solution should have worked, but it's really my bad, so sorry! Try downloading night.xpi again

**Comment by Sndrland** [comment]
I posted this on the zotero subreddit, but I haven't gotten any responses, so I thought to ask on the actual addon repository: 

"I downloaded Zotero ~10 minutes ago, so I'm brand new to the application.

I tried to install zotero night because I want a dark mode and it is nord themed (like the rest of my main programs. The addon looks gorgeous) but it doesn't do anything. I just installed Zotero v6.0.26, added zotero night v0.4.23 as an xpi on the addon menu, it showed up on the list of addons, the program prompted me to restart, I restarted, and then nothing changed. It is still bright white. I followed the example video on the add-on readme page on github, and nothing happened.

I saw online that there some people got errors installing the addon because that version wasn't packed properly, but the dev replied to them saying it's fixed. Also, unlike those people, I don't get any error pop-up. It just doesn't do anything. Zotero thinks everything is working as intended.

I may be completely incompetent and missing something, but I don't know what that may be. My dark-mode-enjoying retinas would greatly appreciate any assistance."

To add to this, after posting on Reddit, I installed Zotero onto a different computer to see if it was just a device problem. It also didn't work on the other device, so Idek what the problem is.

**Reply by Sndrland** [reply]
I was given a fix on Reddit. When the video demonstration restarts zotero, it defaults to light mode. However, on a fresh install, you have to press the little sun button on the top right. That would've been great to know in the documentation, but I have the beautiful dark mode now, so I can't complain.

**Reply by Davo00** [reply]
@JoshSunderland Thanks man, I couldn't find that either

**Reply by hijackso** [reply]
This is huge. Thanks for your response.

**Comment by annawawi** [comment]
please!!! make the theme background dark @tefkah

**Reply by tefkah** [reply]
I am not sure what you mean, could you explain a bit more?

**Reply by annawawi** [reply]
Update UI background which is blue to dark theme.

**Comment by tefkah** [comment]
Hi All!

I just released an alpha version of the plugin that works with Zotero 7 here: https://github.com/tefkah/zotero-night/releases/tag/0.1.0-0

Please check it out and let me know what you think ❤️

**Reply by annawawi** [reply]
the file cannot be used! @tefkah

**Reply by tefkah** [reply]
are you using zotero 7? It is not compatible with zotero 6

**Reply by annawawi** [reply]
Has zotero 7 been released yet? @tefkah

**Reply by tefkah** [reply]
The beta has been released, you can download it here https://www.zotero.org/support/beta_builds

Do note that a lot of other extensions might not work yet.

**Comment by jsthiele** [comment]
Hi, I just tried downloading the xpi file and my browser won't download because it says the file is corrupt. Not sure if this has happened to anyone else.

**Reply by savvycodings** [reply]
yes also getting this issue

**Comment by Electric-V** [comment]
My Zotero is version 6.0.30, and I downloaded and installed Night for Zotero 0.4.23. The page UI can turn black, but the PDF has not been changed, it is still the original white. There is no brightness adjustment button on the right side of the color selection button. What is the reason?thank you!

**Reply by Electric-V** [reply]
Problem solved! This is not a function of the 【Night 】 plugin I'm talking about, it's in the 【Style】 plugin, downloaded the latest version of the Style plugin and you're good to go.

**Comment by JoaoVictorNH** [comment]
Would your addon be able to make the background of the pdf translate addon sidebar ( https://github.com/windingwind/zotero-pdf-translate ) for darkmode?

![example](https://github.com/tefkah/zotero-night/assets/61544856/b6ae0022-0da6-4489-bfdd-9d6675bf33b4)
